### PR TITLE
[HSEARCH] 6.0.0.Beta11

### PR DESCRIPTION
**Pull request by yrodiere**
https://staging.in.relation.to/2020/10/09/hibernate-search-6-0-0-Beta11/

### CODE REVIEW

File: posts/Yoann/2020-10-09-hibernate-search-6-0-0-Beta11.adoc [hunk 1]
```diff
@@ -0,0 +1,233 @@
...
+:hsearch-jira-url-prefix: https://hibernate.atlassian.net/browse
+:hsearch-jira-project-id: 10061
+:hsearch-jira-version-id: 31875
+---
+
+We just published Hibernate Search 6.0.0.Beta11.
+
+This release mainly brings a default analyzer,
+a way to limit automatic reindexing to same-entity updates only,
+a new implementation for AWS IAM authentication,
```
Comment (fax4ever) [inline posts/Yoann/2020-10-09-hibernate-search-6-0-0-Beta11.adoc:16]: maybe we don't need the comma just before the following and

File: posts/Yoann/2020-10-09-hibernate-search-6-0-0-Beta11.adoc [hunk 2]
```diff
@@ -0,0 +1,233 @@
...
+For more information about migration and what we intend to do to help you, see
+https://hibernate.org/search/documentation/migrate/6.0/[the migration guide].
+====
+
+== What's new
+
+[[default-analyzer]]
+=== Default analyzer
+
+Both the Lucene and Elasticsearch backend now have a default analyzer definition,
```
Comment (fax4ever) [inline posts/Yoann/2020-10-09-hibernate-search-6-0-0-Beta11.adoc]: I'm not sure... maybe s/Elasticsearch backend/the Elasticsearch backends.

File: posts/Yoann/2020-10-09-hibernate-search-6-0-0-Beta11.adoc [hunk 3]
```diff
@@ -0,0 +1,233 @@
...
+It is now possible to configure the default value of the `dynamic` attribute of Elasticsearch mappings.
+See link:{hsearch-doc-url-prefix}#backend-elasticsearch-mapping-dynamic[this section of the documentation].
+* link:{hsearch-jira-url-prefix}/HSEARCH-3243[HSEARCH-3243]:
+Hibernate Search will now reject identifier/value bridges whose compatibility
+with the bound property cannot be validated.
+As a rule of thumb, when implementing identifier/value bridges,
+the generic type parameter `I` or `T` should always be set to a raw class.
+For more advanced implementations, you should rely on an `IdentifierBinder` or `ValueBinder`.
+* link:{hsearch-jira-url-prefix}/HSEARCH-3864[HSEARCH-3864]:
+`java.lang.Error` thrown during mass indexing are no longer swallowed and are propagated to the caller.
```
Comment (fax4ever) [inline posts/Yoann/2020-10-09-hibernate-search-6-0-0-Beta11.adoc]: Not sure about. `java.lang.Error`s ? Or change s/are/is.

File: posts/Yoann/2020-10-09-hibernate-search-6-0-0-Beta11.adoc [hunk 4]
```diff
@@ -0,0 +1,233 @@
...
+* link:{hsearch-jira-url-prefix}/HSEARCH-3997[HSEARCH-3997]:
+Mapping arrays of primitive types (int[]/float[]/etc.) now works as expected:
+it no longer leads to a `ClassCastException` upon indexing.
+* link:{hsearch-jira-url-prefix}/HSEARCH-3998[HSEARCH-3998]:
+Mass indexing a (non-abstract) parent class in a type hierarchy
+with a subclass annotated with `@Indexed(enabled = false)` will now correctly ignore that subclass.
+* link:{hsearch-jira-url-prefix}/HSEARCH-3999[HSEARCH-3999]:
+Deleting an entity whose elementCollection is indexed-embedded in another entity
```
Comment (fax4ever) [inline posts/Yoann/2020-10-09-hibernate-search-6-0-0-Beta11.adoc]: Maybe we can put elementCollection between apostrophes.

**Review by fax4ever — APPROVED** [review]
Great job. IMO the post is crystal clear about the news of this release.
I've just few questions. None of them are vital.

File: posts/Yoann/2020-10-09-hibernate-search-6-0-0-Beta11.adoc (hunk 1 above)
Comment (yrodiere) [reply posts/Yoann/2020-10-09-hibernate-search-6-0-0-Beta11.adoc:16]: Both styles are correct, though personal preferences vary. It's called the [Oxford comma](https://www.grammarly.com/blog/what-is-the-oxford-comma-and-why-do-people-care-so-much-about-it/).

**Comment by yrodiere** [comment]
@fax4ever I addressed your comments, thanks.

**Pull request merged**